android/src/main/java/com/mobilepaymentssdkreactnative/MobilePaymentsSdkReactNativeModule.kt

```diff
@@ -20,11 +21,15 @@ import com.squareup.sdk.mobilepayments.payment.PaymentHandle.CancelResult.CANCEL
 import com.squareup.sdk.mobilepayments.payment.PaymentHandle.CancelResult.NOT_CANCELABLE
 import com.squareup.sdk.mobilepayments.payment.PaymentHandle.CancelResult.NO_PAYMENT_IN_PROGRESS
 
+import android.util.Log
```

Comment:
Do you want this?  It might help the developer, so I'm not 100% opposed... but Square will never see that, so it doesn't help _us_ on support.